src/pages/navbar/Navbar.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import "./Navbar.css";
+
+function Navbar({ isLoggedIn, username }) {
+  const [search, setSearch] = useState("");
+
+  return (
+    <header className="navbarContainer">
+      <div className="logo">
+        <img src="" alt="logo" />
+      </div>
+
+      <div className="navigation">
+        <ul className="menu">
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="/topics">Topics</Link>
+          </li>
+          <li>
+            <Link to="/solutions">Solutions</Link>
+          </li>
+        </ul>
+      </div>
+
+      <div className="searchBar">
```

Comment:
Hey! I agree, we can leave it for now but it also seems like a stretch goal. Perhaps searching through deck titles? assuming a user has several pages worth of decks.

Reply:
yes - I think searching through decks is a great feature! I would be inclined to put the search bar in the home page instead of in the navbar in that case but we can figure that out later.